content/en/blog/_posts/2022-02-17-updated-dockershim-faq.md

```diff
@@ -200,6 +200,12 @@ discussion of the changes.
 
 [dep]: https://dev.to/inductor/wait-docker-is-deprecated-in-kubernetes-now-what-do-i-do-e4m
 
+### Is there any tooling that can help me find Dockershim in use
```

Comment:
nit: 
```suggestion
### Is there any tooling that can help me find dockershim in use
```

Reply:
I'm not sure why it would be different here. Why capitalize it?